apps/emqx_bridge_rabbitmq/src/emqx_bridge_rabbitmq_connector.erl

```diff
@@ -43,6 +43,8 @@
 %% Internal callbacks
 -export([publish_messages/3]).
 
+namespace() -> "rabbitmq".
```

Comment:
```suggestion
namespace() -> "bridge_rabbitmq".
```
?

Reply:
i don't think we will have rabbitmq in any context other than bridge.